doc/nrf/device_guides/working_with_nrf/nrf70/gs.rst

```diff
@@ -1,16 +1,25 @@
 .. _nrf7002dk_nrf5340:
 
-Getting started with nRF7002 DK
-###############################
+Getting started with nRF70 Series
+#################################
 
 .. contents::
    :local:
-   :depth: 2
+   :depth: 4
+
+This section gets you started with your nRF70 Series :term:`Development Kit (DK)` using the |NCS|.
```

Comment:
I think this intro text needs to be more generic. It covers all evaluation boards - not just the DK. 

This exact text could potentially be reused in the DK section.